duckduckgo/iOS
# fix deprecation warnings

Task/Issue URL: https://app.asana.com/0/276630244458377/660421450207165
Tech Design URL:
CC:

**Description**:

Now that we have Xcode 9.3 on the MacMini we can fix the deprecation warnings in the project.

**Steps to test this PR**:
1. Check privacy dashboard shows number of trackers as expected
1. Check opening URLs works as expected

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
